tests/run/cpp_stl_string_cpp20.pyx

```diff
@@ -0,0 +1,61 @@
+# mode: run
+# tag: cpp, werror, cpp20
+
+from libcpp cimport bool
+from libcpp.string cimport string
+
+b_A = b'A'
+b_F = b'F'
+b_abc = b"ABC"
+b_def = b"DEF"
+
+def test_string_starts_with_char(py_str):
```

Comment:
```suggestion
def test_string_starts_with_char(bytes py_str):
```

I think this'll let you get a char from the string (the "an integer is required" error). Applies to all the rest of the functions too